src/pages/user-profile/UserProfile.jsx

```diff
@@ -2,25 +2,25 @@ import { useCallback } from 'react'
 import { useAppSelector } from '~/hooks/use-redux'
 import { useParams, useSearchParams, useMatch } from 'react-router-dom'
 
-import ProfileInfo from '~/containers/tutor-profile/profile-info/ProfileInfo'
+import ProfileInfo from '~/containers/user-profile/profile-info/ProfileInfo'
 import CompleteProfileBlock from '~/components/complete-profile/CompleteProfileBlock'
-import VideoPresentation from '~/containers/tutor-profile/video-presentation/VideoPresentation'
-import CommentsWithRatingBlock from '~/containers/tutor-profile/comments-with-rating-block/CommentsWithRaitngBlock'
+import VideoPresentation from '~/containers/user-profile/video-presentation/VideoPresentation'
+import CommentsWithRatingBlock from '~/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock'
 import PageWrapper from '~/components/page-wrapper/PageWrapper'
 import Loader from '~/components/loader/Loader'
 import { userService } from '~/services/user-service'
 import useAxios from '~/hooks/use-axios'
-import videoImgProfile from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
+import videoImgProfile from '~/assets/img/user-profile-page/presentationVideoImg.png'
 
 import { profileItems } from '~/components/profile-item/complete-profile.constants'
 import { defaultResponses } from '~/constants'
-import { responseMock } from '~/pages/tutor-profile/constants'
-import AboutTutorBlock from '~/containers/tutor-profile/about-tutor-block/AboutTutorBlock'
+import { responseMock } from '~/pages/user-profile/constants'
+import AboutTutorBlock from '~/containers/user-profile/about-tutor-block/AboutTutorBlock'
```

Comment:
move components and container imports up

Reply:
Rearranged imports